dev/com.ibm.ws.beanvalidation.core/resources/com/ibm/ws/beanvalidation/resources/nls/BVNLSMessages.nlsprops

```diff
@@ -87,3 +87,16 @@ JNDI_NON_JEE_THREAD_CWNBV0006E.explanation=JNDI operations on a java:comp/env na
 JNDI_NON_JEE_THREAD_CWNBV0006E.useraction=Modify the application so that the JNDI operation occurs on a thread associated with a Java EE application component.
 
 # -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the method with the invalid ValidateOnExecution annotation
+# -----------------------------------------------------------------------
+BVKEY_VALIDATE_ON_EXECUTION_NOT_ALLOWED_CWNBV0007E=CWNBV0007E: @ValidateOnExecution is not allowed on methods overriding a superclass method or implementing an interface. Check configuration for {0}
+BVKEY_VALIDATE_ON_EXECUTION_NOT_ALLOWED_CWNBV0007E.explanation= @ValidateOnExecution is not allowed on methods overriding a superclass method or implementing an interface. A method overriding a superclass method or implementing an interface method inherits the behavior of the method it overrides or implements.
```

Comment:
change
on methods overriding
to
on methods that override

change
A method overriding
to
A method that overrides

change
the method it overrides or implements.
to
the method that it overrides or implements.